python/plugins/processing/ui/widgetPredicateSelector.ui

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why a one column list is used? We can save a lot of space by creating two column list.
![2015-01-12-142659_582x429_scrot](https://cloud.githubusercontent.com/assets/776954/5702850/118ded90-9a67-11e4-9db2-46fa17282cf4.png)